doc/user/content/sql/create-source/kafka.md

```diff
@@ -359,7 +359,6 @@ CREATE VIEW jsonified_kafka_source AS
 ```sql
 CREATE SOURCE proto_source
   FROM KAFKA CONNECTION kafka_connection (TOPIC 'test_topic')
-  WITH (cache = true)
```

Comment:
Also couldn't figure out what happened to `cache`! Is it now parsed as `LEGACYWITH`?